bin/elao-command-migration

```diff
@@ -3,6 +3,22 @@
 
 require __DIR__ . '/../vendor/autoload.php';
 
+function includeIfExists(string $file): bool
+{
+    return file_exists($file) && include $file;
+}
+
+if (
+    !includeIfExists(__DIR__ . '/../../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../../autoload.php') &&
+    !includeIfExists(__DIR__ . '/../autoload.php') &&
```

Comment:
- `__DIR__ . '/../../../autoload.php'` => `vendor/elao/command-migration/bin/../../../autoload.php` when the package is installed on a project in your vendors
- `__DIR__ . '/../vendor/autoload.php'` => `bin/../vendor/autoload.php` when trying to execute the binary from the checkout repository

I think that's only the two required paths actually

Reply:
Thanks @ogizanagi, fixed according to your proposal